plugin/src/main/java/at/helpch/chatchat/ChatChatAPIImpl.java

```diff
@@ -0,0 +1,47 @@
+package at.helpch.chatchat;
+
+import at.helpch.chatchat.api.ChatChatAPI;
+import at.helpch.chatchat.channel.ChannelTypeRegistryImpl;
+import at.helpch.chatchat.hooks.HookManagerImpl;
+import at.helpch.chatchat.mention.MentionManagerImpl;
+import at.helpch.chatchat.placeholder.MiniPlaceholderManagerImpl;
+import at.helpch.chatchat.rule.RuleManagerImpl;
+import at.helpch.chatchat.user.UsersHolderImpl;
+import org.jetbrains.annotations.NotNull;
+
+public class ChatChatAPIImpl implements ChatChatAPI {
+
+    private final ChatChatPlugin plugin;
+
+    public ChatChatAPIImpl(ChatChatPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    public @NotNull UsersHolderImpl usersHolder() {
```

Comment:
not that it *really* matters here, but change the return type to the interface here instead of the concrete class. its just a good habit to have